src/transformation/transformation.ts

```diff
@@ -57,7 +57,7 @@ export class NodeFactory<Source, Output extends Node> {
             const sourcePropName : string | undefined = Object.keys(src).find(e => e == elem);
 
             if (!sourcePropName)
-                throw new Error(`${elem} not found in ${src} (class: ${Object.getPrototypeOf(src).constructor.name})`)
+                return undefined;
```

Comment:
I did some extra checks:

```
class Foo { a; }
Object.keys(new Foo())
=> Array [ "a" ]
foo = new Foo()
=> Object { a: undefined }
delete foo.a
=> true
foo
=> Object {  }
Object.keys(foo)
=> Array []
```

So it looks like that, at least in my current browser, Object.keys is able to distinguish between "undefined" and "missing". So maybe we could return undefined but issue a warning that a missing property has been referenced, this could help detect typos. If it's easy to issue a warning at that point, otherwise, let's just leave this as a note for the future.